routes/transaction.go

```diff
@@ -25,9 +25,21 @@ import (
 	"github.com/pkg/errors"
 )
 
+type TxnStatus string
+
+const (
+	TxnStatusUnconfirmed TxnStatus = "unconfirmed"
```

Comment:
Should we change this from "unconfirmed" to "mempool"? This makes the distinction more clear if we later introduce an "in block" status which really would be better named "unconfirmed", but maybe that would be better as "unconfirmed blocks" or something.

Reply:
+1 to this. We have three statuses in the happy path for the life cycle of a txn: "InMempool", "Unconfirmed", "Committed". 

It's good to disambiguate them now.